.github/workflows/unit-tests.yml

```diff
@@ -49,7 +49,7 @@ jobs:
             npm run test
           dir: frontend
           sonar_args: >
-            -Dsonar.exclusions=**/coverage/**,**/examples/**
+            -Dsonar.exclusions=**/coverage/**,**/examples/**,**/pages/**
             -Dsonar.organization=bcgov-sonarcloud
```

Comment:
ignore the frontend pages folder for now for unit test, cause the UI will be completely different, add this back when we get UI from ux designers